### ISSUE TITLE
Icon request: icon-processing

### ISSUE THREAD

**Issue by Berkmann18** [ForkAwesome/Fork-Awesome#97]
Processing is an IDE with its own languages which is more Java-like (and supports JS and Android oriented Java) which can be used for data science, games, graphics and more.
![Processing](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Processing_3_logo.png/180px-Processing_3_logo.png)

**Comment by xuv** [comment]
Here also, we will need to find and icon in black And white only before being able to add this to the icon set

**Comment by Berkmann18** [comment]
I haven't found any black & white versions even with the related p3.js library.

**Comment by xuv** [comment]
Yes. I don't think there are any. We could make up one, maybe, by just drawing a black p3 in a circle. But that is another story.

**Comment by Berkmann18** [comment]
I guess, the black disked white p3 would be recognisable enough.
